### DISCUSSION TITLE
Using Tabler as UI

### DISCUSSION THREAD

**Discussion by clemone210** [qdm12/ddns-updater#196] [Ideas]
Hi,

would'nt it be nice to use tabler as the furute UI?
It looks clean and nice.
Nginx proxy manager is using it as well.

https://github.com/tabler/tabler

**Comment by qdm12** [comment]
UI will be Flutter based actually, see #191 and #67 (or subscribe to them). That way there should be not only a web ui but also a mobile app. I'm leaving my job at the end of the month to focus full time on open source, so you should expect to have a new UI next month 😉

EDIT: But thanks for the suggestion!
